warehouse/accounts/security_policy.py

```diff
@@ -59,28 +59,31 @@ def _basic_auth_check(username, password, request):
     if userid is not None:
         user = login_service.get_user(userid)
         is_disabled, disabled_for = login_service.is_disabled(user.id)
```

Comment:
I think this line can move in as well?

Reply:
https://github.com/pypi/warehouse/pull/14449/commits/aaa641314a0edb7ae8a7b1c3d6436b5286c8753d